README.md

```diff
@@ -1,8 +1,26 @@
-# Shimmer - Because Ruby could be more shiny!
+Shimmer - Because Ruby could be more shiny!
+===========================================
 
 Shimmer is a collection of Rails extensions that bring advanced UI features into your app and make your life easier as a developer.
 
-## Features
+Features
+--------
+
+### SCSS Components
+
+Shimmer includes a suite of SCSS components aimed at standardizing your project's style and making your SCSS more maintainable. Two key components of this suite are the Stack Component and the Variable component.
```

Comment:
Shimmer includes a suite of styled components that can be implemented in react and also in plain html, or slim.

There's no Variable component